packages/api/src/beacon/routes/validator.ts

```diff
@@ -715,16 +717,20 @@ export function getReturnTypes(): ReturnTypes<Api> {
     {jsonCase: "eth2"}
   );
 
-  const produceBlockOrContents = WithExecutionPayloadValue(
-    WithVersion<allForks.BeaconBlockOrContents>((fork: ForkName) =>
-      isForkBlobs(fork) ? allForksBlockContentsResSerializer(fork) : ssz[fork].BeaconBlock
+  const produceBlockOrContents = WithConsensusBlockValue(
```

Comment:
just modify/rename `WithExecutionPayloadValue` to `WithBlockValues` which handles both vals rather than introducing another level of wrapping around